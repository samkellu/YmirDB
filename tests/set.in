set a 123
list entries
list keys
set b 22 a
list entries
list keys
bye
