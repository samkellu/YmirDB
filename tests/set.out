> not permitted

> no such key

> ok

> []

> ok

> [123 33]

> ok

> [a 22]

> [123 33]

> bye
